bsmetadata/preprocessing_utils.py

```diff
@@ -52,3 +54,40 @@ def preprocess(self, examples: Dict[str, List]) -> Dict[str, List]:
     def _extract_timestamp_from_url(self, url: str) -> Optional:
         # This would have to be implemented.
         return None
+
+
+class HtmlPreprocessor(MetadataPreprocessor):
+    """todo"""
```

Comment:
I really more doc string here to understand the code. 😅

Reply:
This is indeed an oversight. I have added it. I'm at your disposal if you need more.